tfa_support/sh_tfa_support.lua

```diff
@@ -24,6 +24,21 @@
 
 local PLUGIN = PLUGIN
 
+--[[
+DoAutoCreation:
+
+If `true`:
+	Auto-generate items for all weapons with `tfa_` prefix
+
+If `false`:
+	Auto-generate only that items that described in `sh_tfa_weps.lua` file
+
+*Black List works for both methods
+]]
+PLUGIN.DoAutoCreation = true
```

Comment:
You might want to document this info inside of `PLUGIN.readme` so people can read it on the site

Reply:
Updated